apps/e2e/forms-storybook/src/app/checkbox/checkbox.component.html

```diff
@@ -196,14 +196,10 @@
   <sky-row class="sky-padding-even-md">
     <sky-column>
       <div class="sky-switch-icon-group">
-        <sky-checkbox icon="bold" label="bold" [checked]="true" />
-        <sky-checkbox icon="umbrella" label="umbrella" [checked]="false" />
-        <sky-checkbox icon="star" label="favorite" [checked]="false" />
-        <sky-checkbox
-          icon="exclamation-triangle"
-          label="careful"
-          [checked]="false"
-        />
+        <sky-checkbox iconName="text-bold" label="bold" [checked]="true" />
```

Comment:
I think a lot of the changes in this file can just be deleted instead? Most already have an SVG equivalent below them, so the screenshot now has a lot of duplicates

Reply:
Fixed.